packages/@storefront/breadcrumbs/src/refinement-crumbs/index.ts

```diff
@@ -20,6 +20,10 @@ class RefinementCrumbs {
     this.updateState();
   }
 
+  shouldUpdate() {
```

Comment:
Is there a way we can use the `shouldUpdate` params to determine whether or not an update is required?

ie. can we compare the indices of the currently `selected` refinements (for this navigation) with the previously selected indices?